### ISSUE TITLE
How to debug Gadget Helper.gs?

### ISSUE THREAD

**Issue by distributev** [distributev/gmail-wordpress-mailchimp-integration#15]
I find it difficult to trace what is happening there. There are few Logger.log statements but the View Logs screen it's always empty no matter what I do in Gmail. I don't even know if this code is executed or not.

The only thing I manage to do was using the Chrome developer tools console and saw some things there. 

How I can debug Gadget Helper.gs live?

**Comment by mdkass** [comment]
I'm going to report logs to a Google spreadsheet.

**Comment by mdkass** [comment]
I've added a new function called addLog and there are 2 examples how to use it in the saveEvent function.
You can see the log info in this spreadsheet:
https://docs.google.com/spreadsheets/d/1_xUj5zizYTjrmP5xv2ZjMIV2C54Wjmd6uPcZMN4cOIY/edit?usp=sharing

**Comment by distributev** [comment]
This is good.

**Comment by mdkass** [comment]
just a reminder: Please don't forget to deploy the web app after each edit.

**Comment by distributev** [comment]
Good catch. I was not doing that but it was looking as my chances take effect.
